chia/cmds/cmds_util.py

```diff
@@ -10,3 +26,48 @@ def transaction_submitted_msg(tx: TransactionRecord) -> str:
 
 def transaction_status_msg(fingerprint: int, tx_id: bytes32) -> str:
     return f"Run 'chia wallet get_transaction -f {fingerprint} -tx 0x{tx_id}' to get status"
+
+
+async def check_client_connection(rpc_client: RpcClient, node_type: str, rpc_port: int) -> bool:
+    from aiohttp import ClientConnectorError
+
+    try:
+        await rpc_client.healthz()
+    except ClientConnectorError:
+        print(f"Connection error. Check if {node_type.replace('_', ' ')} rpc is running at {rpc_port}")
+        print(f"This is normal if {node_type.replace('_', ' ')} is still starting up")
+        return False
+    return True
+
+
+async def execute_with_any_node(
```

Comment:
I would rename to `execute_with_any_service`. Node means fulls node in chia vocabulary.

Reply:
Renamed!